config.libsonnet

```
@@ -98,5 +98,11 @@
     // This list of disk device names is referenced in various expressions.
     diskDevices: ['mmcblk.p.+', 'nvme.+', 'rbd.+', 'sd.+', 'vd.+', 'xvd.+', 'dm-.+', 'dasd.+'],
     diskDeviceSelector: 'device=~"%s"' % std.join('|', self.diskDevices),
+
+    // Certain workloads (e.g. KubeVirt/CDI) will fully utilise the persistent volume they claim
+    // the size of the PV will never grow since they consume the entirety of the volume by design.
+    // This selector allows an admin to 'pre-mark' the PVC of such workload so specific storage alerts will not fire.
+    // With the default selector, adding a label `fully-utilised: 'true'` to the PVC will have the desired effect.
+    pvcFullyUtilisedSelector: 'label_fully_utilised="true"',
```

This can be used for multiple scenarios as a generic exclusion of the particular PV. Let's change the selector name to reflect that. Maybe sth like the following?

```suggestion
    pvExcludedSelector: 'excluded_from_alerts="true"',
```

---

makes sense. I will prefix it with `label_` since this is how ksm transforms labels to metric names